test/unit/from-api-elements/compile-test.coffee

```diff
@@ -194,9 +206,130 @@ describe('compileFromApiElements()', ->
     )
   )
 
+  describe('API description causing a \'not specified in URI Template\' warning', ->
+    # The warning was previously handled by compiler, but now parser already
+    # provides the same kind of warning.
+
+    err = undefined
+    warnings = undefined
+    transactions = undefined
+
+    beforeEach((done) ->
+      compile('''
+        FORMAT: 1A
+        # Beehive API
+        ## Honey [/honey]
+        ### Remove [DELETE]
+        + Parameters
+            + beekeeper: Honza (string)
+        + Response 203
+      ''', (args...) ->
+        [err, {warnings, transactions}] = args
+        done()
```

Comment:
The problem is, when I test errors, the first error gets returned also in the first parameter (`err`), so I couldn't test the results.

But that doesn't apply for tests of warnings. I could change it for warnings and add explanation for errors.